source/rules/roles.txt

```diff
@@ -197,14 +219,14 @@ runs the query.
 
 For each document returned by the query, your App evaluates possible
 roles in :ref:`role order <role-order>` and assigns the first applicable
-role, if any. A role applies to a given document if its :ref:`Apply When expression 
-<apply-when-expressions>` evaluates to ``true`` when run against
+role, if any. A role applies to a given document if its "apply when"
+:ref:`expression <expressions>` evaluates to ``true`` when run against
 the document.
 
 The set of roles that a user can use for a given request depends on the
 collection they're accessing. If the collection has a set of
-collection-level roles defined, then a user can use any of those roles.
-If there are no collection-level rules, the request falls back to the
+collection-level roles defined, then a user may play any of those roles.
```

Comment:
I'm curious about the wording change here. "use" doesn't seem accurate, but "play" seems odd too. I've typically seen users and roles described in a couple of ways:
- the user is the role: a user may **be** any of those roles
- the user has a role: a user may **have** any of those roles

Reply:
Yes! I inherited this verbiage, and I fixed it, and it seems my fix was undone in a merge. I will re-apply.